scripts/dr-bench.py

```diff
@@ -76,10 +76,41 @@ def mhp(ctx, bench, fork, nprocs, sycl_cpu, sycl_gpu):
     command = (
         f'{env} mpirun {"-launcher=fork" if fork else ""} -n {nprocs} '
         f'{bench} {sycl_args} {ctx.obj["COMMON_ARGS"]} '
-        f'--benchmark_out={ctx.obj["OUTPUT"]}'
+        f'--benchmark_out=mhp_{nprocs}_{ctx.obj["OUTPUT"]}'
     )
     execute(command, ctx)
 
+    plot_command = 'python Plotter.py --mhp'
```

Comment:
This command does not work, but silently pass despite of
`python: can't open file '/home/u101139/github-runners/_work/distributed-ranges/distributed-ranges/build/benchmarks/gbench/mhp/Plotter.py': [Errno 2] No such file or directory` error (see CI output in icpx)

To avoid silent pass add `check=True` to `subprocess.run` in `execute` function.

Reply:
Not applicable to my pr anymore I think